telegram-purple.nsi

```diff
@@ -92,23 +92,23 @@ Section "MainSection" SEC01
 	
 	SetOutPath "$PidginDir\locale"
 	File /nonfatal "/oname=bg\LC_MESSAGES\telegram-purple.mo" "po\bg.mo"
-	File /nonfatal "/oname=cs_CZ\LC_MESSAGES\telegram-purple.mo" "po\cs_CZ.mo"
-	File /nonfatal "/oname=de_DE\LC_MESSAGES\telegram-purple.mo" "po\de_DE.mo"
+	File /nonfatal "/oname=cs\LC_MESSAGES\telegram-purple.mo" "po\cs.mo"
+	File /nonfatal "/oname=de\LC_MESSAGES\telegram-purple.mo" "po\de.mo"
+	File /nonfatal "/oname=es\LC_MESSAGES\telegram-purple.mo" "po\es.mo"
 	File /nonfatal "/oname=es_AR\LC_MESSAGES\telegram-purple.mo" "po\es_AR.mo"
 	File /nonfatal "/oname=fr\LC_MESSAGES\telegram-purple.mo" "po\fr.mo"
-	File /nonfatal "/oname=it_IT\LC_MESSAGES\telegram-purple.mo" "po\it_IT.mo"
+	File /nonfatal "/oname=it\LC_MESSAGES\telegram-purple.mo" "po\it.mo"
 	File /nonfatal "/oname=nl\LC_MESSAGES\telegram-purple.mo" "po\nl.mo"
-	File /nonfatal "/oname=pl_PL\LC_MESSAGES\telegram-purple.mo" "po\pl_PL.mo"
+	File /nonfatal "/oname=pl\LC_MESSAGES\telegram-purple.mo" "po\pl.mo"
 	File /nonfatal "/oname=ru_RU\LC_MESSAGES\telegram-purple.mo" "po\ru_RU.mo"
 	File /nonfatal "/oname=sq\LC_MESSAGES\telegram-purple.mo" "po\sq.mo"
 	File /nonfatal "/oname=uk\LC_MESSAGES\telegram-purple.mo" "po\uk.mo"
 	
 	SetOutPath "$PidginDir"
-	File "${WIN32_DEV_TOP}\libgpg-error-1.12-2\bin\libgpg-error-0.dll"
-	File "${WIN32_DEV_TOP}\libgcrypt-1.6.3\bin\libgcrypt-20.dll"
-	File "${WIN32_DEV_TOP}\libwebp-0.4.3-1\bin\libwebp-5.dll"
-	File "${WIN32_DEV_TOP}\mingw\bin\libgcc_s_sjlj-1.dll"
-	
+	File "${WIN32_DEV_TOP}\libgpg-error-0.dll"
+	File "${WIN32_DEV_TOP}\libgcrypt-20.dll"
+	File "${WIN32_DEV_TOP}\libwebp.dll"

```

Comment:
Good catch, but nope, I build libweb-6.dll, and since it's custom-built I thought I'd leave the 6 away.

Thinking about it again, I *really* should add the 6 back in.

This review is going awesome, thank you :D